src/ImportDefinitionsBundle/Interpreter/ObjectResolver.php

```diff
@@ -47,8 +47,25 @@ public function interpret(
         $listing->setUnpublished($configuration['match_unpublished']);
         $found = $listing->count();
 
-        if ($found < 1 || $found > 1) {
-            // too few or too many found
+        if ($found < 1) {
+            // too few found
+            if (array_key_exists('create_missing',$configuration) && $configuration['create_missing']) {
+                $parent = \Pimcore\Model\DataObject::getByPath($configuration['object_path']);
+                $key = \Pimcore\Model\Element\Service::getValidKey($value, 'object');            
+                $object = new $class();
+                $setter = 'set' . ucfirst($configuration['field']);
+                $object->$setter($value);
```

Comment:
not sure about this. A valid use case is for a product import with categories, manufacturers, etc. and only setting one value is not enough for that.